.github/CODEOWNERS

```diff
@@ -128,7 +128,7 @@
 /core/controllers/contributor_dashboard*.py @oppia/contributor-experience-team
 /core/domain/opportunity*.py @oppia/contributor-experience-team
 /core/domain/translation*.py @oppia/contributor-experience-team
-/core/storage/translation/ @oppia/data-and-stability-reviewers
+/core/storage/translation/ @oppia/contributor-experience-team
```

Comment:
I think core/storage is actually owned by the core-reviewers team so it doesn't need to be here at all?

Reply:
Done.